config/metric_config_sample.yaml

```diff
@@ -126,3 +126,21 @@ output:
       # Since segment limits the number of events per second, we need to sleep and write in small enough batches
       sleep: 1000
       batchSize: 30
+  - dataFrameName: table1
+    outputType: Elasticsearch
+    outputOptions:
+      # The index name to write to
+      resource: index1
+      # type
+      indexType:
+      # The saveMode for the write operation
+      saveMode: Append
+      # The write Operation to preform (index(default)/create/upsert/update)
```

Comment:
index(default)?

Reply:
it's the default value for writeOperation. I can remove it if it's confusing.